tests/python/ci/test_ci.py

```diff
@@ -39,6 +39,7 @@ def run(self, *args, **kwargs):
         return proc
 
 
+@pytest.mark.skipif(os.getenv("WHEEL_TEST") is not None, reason="Test not supported in wheel.")
```

Comment:
can we de-duplicate all of these into a utils decorator
```suggestion
@tvm.testing.skip_in_wheel_test
```